templates/product/wishlist/_add_remove_button.html.twig

```diff
@@ -0,0 +1,5 @@
+{% if product.id in ids_on_wishlist %}
```

Comment:
the template name _add_remove_button.html.twig is misleading, because here you are adding a button to remove OR add to wishlist. Please change it to something more specific.